src/dune

```diff
@@ -1,10 +1,28 @@
+;; `melange.emit` is a Dune stanza that will produce build rules to generate
+;; JavaScript files from sources using the Melange compiler
+;; https://dune.readthedocs.io/en/stable/melange.html#melange-emit
 (melange.emit
+
+ ;; The `target` field is used by Dune to put all JavaScript artifacts in a specific folder inside `_build/default`
+ ;; This field `target` allows to have multiple `melange.emit` stanzas in the `dune` file
+ ;; if, for example, we want to produce ES6 and CommonJS output from the same sources.
  (target output)
+
+ ;; The `alias` field can be used later on to build just the artifacts from this `melange.emit`
+ ;; by calling `dune build @react`
  (alias react)
+
+ ;; Dune will generate rules to generate JavaScript targets from the libraries sources as well
  (libraries reason-react world)
```

Comment:
I would phrase it differently, like:

"here's the list of dependencies of the stanza". In this case (being a melange.emit), dune will look into those dependencies and generate JavaScript targets from those libraries as well. 

Caveat: those libraries need to be specified with `(modes melange)`

Reply:
https://github.com/melange-re/melange-opam-template/pull/26/commits/9ade436272020301a0278344fe06ea9016e4e3c9